sections/02_intro.md

```diff
@@ -1,97 +1,155 @@
+also new challenges. We expect that algorithms to automatically extract
+meaningful patterns and provide sufficient context to enable us to act will be
+required.
 
-One potential future that we could imagine is a world in which data, once
-gathered, is rapidly integrated into our understanding of living systems. What
-we learn is rapidly put to use. Our health-related activities are constantly
-monitored (e.g. by wearables) and all of our interactions with health care
-systems are extensively tracked. These sources of information are combined to
-help to guide our health care and maintenance. We'd be able to compare our state
-and trajectory to (anonymized) others, and identify means to improve our health.
-These means might contain drug combinations selected based on personalized
-predictions.
+Concurrent with this explosive growth in biomedical data, a new class of machine
+learning algorithm has become widespread in the domain of image analysis.
+Computer scientists are now building many-layered neural networks from
+collections of millions of images. In a famous example, scientists from Google
+demonstrated that a neural network could learn to identify cats simply by
+watching online videos [@doi:10.1109/ICASSP.2013.6639343]. Such approaches,
+termed deep learning, seem like a solution to the challenge presented by the
+growth of data in biomedicine. Perhaps these algorithms could identify the
+biological "cats" hidden in our data - the patterns that exist but that we don't
+know to look for - and could act on them.
 
-### If this happens, is deep learning required for any of it? Are we any closer
-### because of the advent of deep learning?
+Deep learning has transformed image analysis, but researchers' initial forays
+into the use of these techniques in biomedicine have been relatively limited.
+There are certainly numerous promising examples, but we have not yet seen the
+massive convergence on these approaches that occurred in the field of image
+analysis. In this review, we discuss whether this is simply a matter of time or
+if there are unique challenges posed by biomedical data that render deep
+learning methods more challenging or less fruitful to apply.
 
-*"Categorize" and "treat" sound a bit like PMI goals. Another way to think about
-this would be: do we think that deep learning will make much of a difference
-for the precision medicine initiative (PMI)?*
+### What is deep learning?
 
-### General deep learning introduction
+Deep learning is built on a biologically-inspired approach from machine learning
+termed neural networks. Each neuron in a computational neural network, termed a
+node, has inputs, an activation function, and outputs. Each value from the
+inputs is usually multiplied by some weight and combined and summarized by the
+activation function. The value of the activation function is then multiplied by
+another set of weights to produce the output `TODO: we probably need a figure
```

Comment:
Does anybody know the people from @asimovinstitute who put the nn zoo together? I would be more comfortable citing a version of the figure deposited in zenodo or figshare than a webage that could change. The metadata for zenodo or figshare could provide a link to the webpage.

It might also be great to see if they want to participate in this specific aspect (or more broadly). I agree with @gwaygenomics that we don't want to dive deeply into specific architectures at this time.